pkg/api/validation/validation.go

```diff
@@ -49,6 +49,7 @@ var dns1123LabelErrorMsg string = fmt.Sprintf(`must be a DNS label (at most %d c
 var dns952LabelErrorMsg string = fmt.Sprintf(`must be a DNS 952 label (at most %d characters, matching regex %s): e.g. "my-name"`, util.DNS952LabelMaxLength, util.DNS952LabelFmt)
 var pdPartitionErrorMsg string = intervalErrorMsg(0, 255)
 var portRangeErrorMsg string = intervalErrorMsg(0, 65536)
+var portStringErrorMsg string = "port string must be compliant with https://tools.ietf.org/html/rfc6335#section-5.1"
```

Comment:
ContainerPort.name is validated as https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/validation/validation.go#L595 DNS_LABEL